docs/src/documentation/03-components/01-action/button/01-guidelines.mdx

```diff
@@ -36,7 +36,9 @@ check out our [interactive guide on action components](/guides/using-button-type
 
 > "If everything looks important, then nothing is important."
```

Comment:
If we're going to the trouble of styling this completely as a quote, it doesn't require the quotation marks as much.
```suggestion
> If everything looks important, then nothing is important.
```